docs/desktop-wpf/migration/differences-from-net-framework.md

```diff
@@ -27,20 +27,13 @@ When your project uses `<PackageReference>`, packages aren't stored locally in a
 
 ## Code Access Security (CAS)
 
-Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. Therefore, WPF has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
+Code Access Security (CAS) isn't supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full trust. Therefore, WPF has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
 
 Publicly defined CAS-related types were moved out of the WPF assemblies and into the CoreFX assemblies. The WPF assemblies have type-forwarding set to the new location of the moved types.
 
 | Source Assembly | Target Assembly | Type                |
 | --------------- | --------------- | ------------------- |
-| **WindowsBase.dll** | **System.Security.Permissions.dll** | <xref:System.Security.Permissions.MediaPermission> |
-|                     |                                     | <xref:System.Security.Permissions.MediaPermissionAttribute> |
-|                     |                                     | <xref:System.Security.Permissions.MediaPermissionAudio> |
-|                     |                                     | <xref:System.Security.Permissions.MediaPermissionImage> |
-|                     |                                     | <xref:System.Security.Permissions.MediaPermissionVideo> |
-|                     |                                     | <xref:System.Security.Permissions.WebBrowserPermission> |
-|                     |                                     | <xref:System.Security.Permissions.WebBrowserPermissionAttribute> |
-|                     |                                     | <xref:System.Security.Permissions.WebBrowserPermissionLevel> |
+| **WindowsBase.dll** | **System.Security.Permissions.dll** | <xref:System.Security.Permissions.MediaPermission> <br /> <xref:System.Security.Permissions.MediaPermissionAttribute> <br /> <xref:System.Security.Permissions.MediaPermissionAudio> <br /> <xref:System.Security.Permissions.MediaPermissionImage> <br /> <xref:System.Security.Permissions.MediaPermissionVideo> <br /> <xref:System.Security.Permissions.WebBrowserPermission> <br /> <xref:System.Security.Permissions.WebBrowserPermissionAttribute> <br /> <xref:System.Security.Permissions.WebBrowserPermissionLevel> |
```

Comment:
I think the file names should be *italic* instead of **bold** as suggested [here](https://github.com/dotnet/docs/blob/master/styleguide/template.md#text-styling)